build.gradle

```diff
@@ -316,10 +317,10 @@ dependencies {
   testImplementation("junit-addons:junit-addons:1.4")
 
   annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
-  annotationProcessor("org.projectlombok:lombok")
-  testAnnotationProcessor("org.projectlombok:lombok")
-  compileOnly("org.projectlombok:lombok")
-  testCompileOnly("org.projectlombok:lombok")
+  annotationProcessor("org.projectlombok:lombok:${lombokVersion}")
+  testAnnotationProcessor("org.projectlombok:lombok:${lombokVersion}")
+  compileOnly("org.projectlombok:lombok:${lombokVersion}")
+  testCompileOnly("org.projectlombok:lombok:${lombokVersion}")
```

Comment:
Use the latest Spring Boot 2.5.2 and you have the latest lombok. We should try to use the lombok bundled with spring boot if possible.

Reply:
Yeah this was an attempt of me hoping something else was causing the not exported errors and trying different things to resolve it. I will change it back.